erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.js

```diff
@@ -21,7 +21,45 @@ frappe.ui.form.on("Payment Reconciliation Payment", {
 				doc: frm.doc,
 				method: 'get_difference_amount',
 				args: {
-					child_row: row
+					child_row: row,
+					source_table : 'payments'
```

Comment:
```suggestion
					source_table: 'payments'
```
@marination looks good just some formatting and minor changes